doc/faq.rst

```diff
@@ -181,86 +181,59 @@ Using Spyder
-       |C:\\Users\\<username>\\Miniconda    | /Users/<username>/opt/miniconda |
-       +------------------------------------+---------------------------------+
-       |C:\\Users\\<username>\\Anaconda3    | /Users/<username>/opt/anaconda3 |
-       +------------------------------------+---------------------------------+
-       |C:\\Users\\<username>\\Miniconda3   | /Users/<username>/opt/miniconda3|
-       +------------------------------------+---------------------------------+
-       |C:\\Anaconda                        |   /opt/anaconda                 |
-       +------------------------------------+---------------------------------+
-       |C:\\Miniconda                       |   /opt/miniconda                |
-       +------------------------------------+---------------------------------+
-       |C:\\Anaconda3                       |   /opt/anaconda3                |
-       +------------------------------------+---------------------------------+
-       |C:\\Miniconda3                      |   /opt/miniconda3               |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Anaconda           |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Miniconda          |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Anaconda3          |                                 |
-       +------------------------------------+---------------------------------+
-       |C:\\ProgramData\\Miniconda3         |                                 |
-       +------------------------------------+---------------------------------+
+   If you want to use other packages in Spyder that don't come with our installer, you need to have your own Python distribution installed; we recommend `Miniconda`_ or another Conda-based option.
+   For Spyder to recognize it automatically, you should use a Conda-based distribution with its default install path.
 
    .. _Miniconda: https://docs.conda.io/en/latest/miniconda.html
 
-   Then, you need to create a new conda environment with the modules that you want to use with Spyder and include ``spyder-kernels`` in it. For example, if you want to use ``scikit-learn``, open your terminal or the Anaconda prompt on Windows and run the following commands:
-
-   .. code-block:: bash
-
-      conda create -n spyder-env -y
-      conda activate spyder-env
-      conda install spyder-kernels scikit-learn -y
-
-   Finally, you need to connect Spyder to this environment by changing Spyder’s default Python interpreter. To do this, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
+   Create a new conda environment with ``spyder-kernels`` and the packages that you want to use installed.
```

Comment:
```suggestion
   Create a new conda environment with ``spyder-kernels`` and the packages that you want to use installed on it.
```

I think it's missing to better understand the meaning of this sentence.

Reply:
Hmm, well that isn't grammatical. I reworded it this way instead:

```suggestion
   Create a new conda environment containing ``spyder-kernels`` and the packages that you want to use.
```